### PR TITLE
Es6 modules

### PR DESCRIPTION
**Pull request by bookaneers**
Fernando Figueiredo | Completed es6-modules
Screenshot (19)
Screenshot (20)

### CODE REVIEW

File: es6-modules/src/components/todo-list.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+const { createElement } = require('../lib');
```
Comment (scbowler) [inline es6-modules/src/components/todo-list.js:1]: You need to go through every file and update all `requires` to be `import` statements

File: es6-modules/src/components/todo-list.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      createElement('ul', { class: 'list-group shadow-sm mb-4' }, todos.map(todo => {
+        return this.renderTodoItem(todo);
+      }))
+    );
+    todoList.addEventListener('change', this.handleChange);
+    return todoList;
+  }
+}
+
+module.exports = TodoList;
```
Comment (scbowler) [inline es6-modules/src/components/todo-list.js:52]: All `module.exports` need to be converted to `export default` statements

**Review by scbowler — CHANGES_REQUESTED** [review]
Make sure to go through ALL the files within the `src` folder and update them to use ES6 Modules. Let me know if you have any questions.

File: es6-modules/src/components/todo-list.js (hunk 1 above)
Comment (bookaneers) [reply es6-modules/src/components/todo-list.js:1]: Fernando Figueiredo | Updated all files in es6-modules.
Screenshot (21)

**Pull request merged**